R/getmask.R

```diff
@@ -0,0 +1,72 @@
+#' Get mask
+#'
+#' @description Build a mask with the intersection of the topography and the spatial inventory.
+#'
+#' @param inventory Input inventory (see the inputs formats and metadata in the
+#'   vignette) (data.frame)
+#'
+#' @param topography Digital terrain model (DTM) of the inventoried plot (LiDAR,
+#'  1m resolution)
+#'  (RasterLayer **with a crs in UTM**) (See \code{\link{DTMParacou}})
+#'  We advise you to generate your raster with Qgis rather than with the
```

Comment:
Why do you suggest to create the raster in QGis ? If you added some code to transform the raster input to the right format, that shouldn't matter no ? And I thought that the package was using the format of the raster package

Reply:
Sorry this is a copy paste from Vincyane functions, but we could remove it (in that case everywhere). To discuss with @VincyaneBadouard .